src/jit/codegenlegacy.cpp

```diff
@@ -19555,16 +19560,17 @@ regMaskTP           CodeGen::genCodeForCall(GenTreePtr  call,
                         regTracker.rsTrackRegTrash(REG_JUMP_THUNK_PARAM);
                         callTypeStubAddr = emitter::EC_INDIR_R;
                         getEmitter()->emitIns_Call(emitter::EC_INDIR_R,
-                                                 NULL,    // methHnd
-                                                 INDEBUG_LDISASM_COMMA(sigInfo)
-                                                 NULL,                  // addr
-                                                 args,
-                                                 retSize,
-                                                 gcInfo.gcVarPtrSetCur,
-                                                 gcInfo.gcRegGCrefSetCur,
-                                                 gcInfo.gcRegByrefSetCur,
-                                                 ilOffset,
-                                                 REG_JUMP_THUNK_PARAM);
+                                                  NULL,    // methHnd
+                                                  INDEBUG_LDISASM_COMMA(sigInfo)
```

Comment:
(Since you are at it).
There are many instances of NULL that could be replaced with nullptr in the code that you have edited.  i will leave it up to you whether you want to do this change.